app/celery/celery.py

```diff
@@ -49,13 +49,14 @@ def init_app(self, app):
             task_cls=make_task(app),
         )
 
-        # Register the xray handlers
-        signals.after_task_publish.connect(xray_after_task_publish)
-        signals.before_task_publish.connect(xray_before_task_publish)
-        signals.task_failure.connect(xray_task_failure)
-        signals.task_postrun.connect(xray_task_postrun)
-        signals.task_prerun.connect(xray_task_prerun)
-        signals.beat_init.connect(xray_task_prerun)
+        if app.config["AWS_XRAY_ENABLED"] == "true":
```

Comment:
To minimize typo and typing errors, we want:
```current_app.config["FRESH_DESK_ENABLED"] is True```

But that would require for the `AWS_XRAY_ENABLED` constant to be declared using the `bool` function of the `environs` module as such:

```
AWS_XRAY_ENABLED = env.bool("AWS_XRAY_ENABLED", False)
```

The [environs module](https://pypi.org/project/environs/) is a nifty library to make configuration parsing more reliable in general.